lib/main.coffee

```diff
@@ -31,9 +26,20 @@ module.exports =
     @subscriptions.add atom.config.observe 'linter-phpmd.rulesets',
       (rulesets) =>
         @rulesets = rulesets
-    @subscriptions.add atom.config.observe 'linter-phpmd.projectRules',
-      (projectRules) =>
-        @projectRules = projectRules
+    if atom.config.get('linter-phpmd.projectRules') == true &&
+    atom.config.get('linter-phpmd.rulesets') ==
+    atom.config.defaultSettings['linter-phpmd'].rulesets
```

Comment:
Although this `defaultSettings` property seems very useful, I'd prefer that this used the public API by calling [`Config::getSchema()`](https://atom.io/docs/api/v1.7.3/Config#instance-getSchema) and using the default property of the returned object.